packages/dds/tree/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -496,12 +485,12 @@ export function splitMarkOnInput<TMark extends InputSpanningMark<unknown>>(
  * @returns A pair of marks equivalent to the original `mark`
  * such that the first returned mark has output length `length`.
  */
-export function splitMarkOnOutput<TMark extends OutputSpanningMark<unknown>>(
+export function splitMarkOnOutput<T, TMark extends OutputSpanningMark<T>>(
```

Comment:
This change might let you remove some of the `as [TMark, TMark]` casts.

Reply:
Fixed